crates/turborepo-lib/src/config.rs

```diff
@@ -23,7 +23,7 @@ pub enum Error {
     #[error(transparent)]
     PackageJson(#[from] turborepo_repository::package_json::Error),
     #[error(
-        "Could not find turbo.json. Follow directions at https://turbo.build/repo/docs to create \
+        "Could not find turbo.json.\nFollow directions at https://turbo.build/repo/docs to create \
```

Comment:
This feels unrelated to deleting Go

Reply:
This was due to an issue with error printing and regexes. Miette line breaks at 80 characters and adds a pipe character. By line breaking early, we avoid the pipe which makes it easier to match on the text. Not needed now if we're stripping out the miette changes for a later PR